http/src/test/scala/org/http4s/blaze/http/http2/CodecUtils.scala

```diff
@@ -6,8 +6,11 @@ import java.util
 import org.http4s.blaze.util.BufferTools._
 
 object CodecUtils {
+
+  def byteData: Array[Byte] = "The quick brown fox jumps over the lazy dog".getBytes()
```

Comment:
Break the habit of using the platform-default encoding for `getBytes`.  It's just a test, but young programmers may be watching and pasting at any time.